packages/askui-nodejs/src/core/model/annotation-result/annotation-interface.ts

```diff
@@ -1,8 +1,10 @@
 import { CustomElementJson } from '../custom-element-json';
+import { DetectedElement } from './detected-element';
 
 export interface AnnotationRequest {
   imagePath?: string,
+  elements?: DetectedElement[]
   outputPath?: string,
-  fileNamePrefix?:string,
+  fileNamePrefix?: string,
```

Comment:
Automated formatting or you having an eye for details? ;) It's important to have an eye for details! Details make the difference between good enough and awesome!

Reply:
auto-formated